src/openforms/authentication/contrib/digid_eherkenning_oidc/models.py

```diff
@@ -81,6 +87,117 @@ def oidc_authentication_callback_url(cls) -> str:  # type: ignore
         return "oidc_authentication_callback"
 
 
+@default_loa_choices(DigiDAssuranceLevels)
+class OFEIDASConfig(BaseConfig):
+    class Meta:
+        verbose_name = _("eIDAS (OIDC)")
+
```

Comment:
please respect the order of attributes in django models. I'll have to enable the `DJ` linting rules soon :-)

https://docs.astral.sh/ruff/rules/django-unordered-body-content-in-model/